src/pages/home/Home.tsx

```diff
@@ -177,19 +180,88 @@ const Home: React.FC = () => {
         <Button>Assinar</Button>
       </Flex>
 
-      <Box
+      <Footer
         sx={{
-          height: '30rem',
-          backgroundColor: 'primary',
-          position: 'relative',
+          height: '11rem',
+          px: 4,
+          py: '1.25rem',
+          backgroundColor: '#222222'
         }}
+        behaviour="relative"
       >
-        <Image
-          sx={{ position: 'absolute', left: 4, bottom: 4 }}
-          width="120"
-          src={icSrc}
-        ></Image>
-      </Box>
+        <FooterRow sx={{ alignItems: 'center' }}>
+          <Box
+            data-testid="footerLeft"
+            sx={{ flexGrow: 1, display: 'inline-block' }}
+          >
+            <Image
+              width="160"
+              sx={{ display: 'inline-block', verticalAlign: 'middle' }}
+              src={whiteLogoSrc}
+            />
+          </Box>
+
+          <Link sx={{ ml: 5, color: '#E0E0E0' }}>Direitos autorais</Link>
+
+          <Link sx={{ ml: 5, color: '#E0E0E0' }}>Termos de uso</Link>
+
+          <Link sx={{ ml: 5, color: '#E0E0E0' }}>Políticas de privacidade</Link>
+        </FooterRow>
+        <FooterRow sx={{ alignItems: 'center', marginTop: 4 }}>
+          <Box
+            data-testid="footerLeft"
+            sx={{ flexGrow: 1, display: 'inline-block' }}
+          >
+            <Link
+              sx={{
+                backgroundColor: '#3F3F46',
+                padding: 1,
+              }}
+            >
+              <AiOutlineFacebook color="#fff" />
+            </Link>
+            <Link
+              sx={{
+                backgroundColor: '#3F3F46',
+                ml: 3,
+                padding: 1,
+              }}
+            >
+              <AiOutlineTwitter color="#fff"  />
+            </Link>
+            <Link
+              sx={{
+                backgroundColor: '#3F3F46',
```

Comment:
Update the themes/index.ts to add the colors there
```suggestion
                backgroundColor: 'colors.`...`',
```